zerver/tests/test_slack_importer.py

```diff
@@ -171,12 +174,16 @@ def test_users_to_zerver_userprofile(self, mock_get_data_file: mock.Mock) -> Non
 
         # Test custom profile fields
         self.assertEqual(customprofilefield[0]['field_type'], 1)
-        self.assertEqual(customprofilefield[1]['name'], 'slack custom field 2')
```

Comment:
Does this slightly lose coverage for checking that the string output still works?

Reply:
Added it back